src/Equinor.ProCoSys.IPO.Infrastructure/Migrations/20210302153400_AddSystemToMcPkgs.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Equinor.ProCoSys.IPO.Infrastructure.Migrations
+{
+    public partial class AddSystemToMcPkgs : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "System",
+                table: "McPkgs",
+                type: "nvarchar(max)",
+                nullable: false,
+                defaultValue: "");
```

Comment:
How will you migrate existing IPO's?

Reply:
Need to discuss this with you and maybe Pål